pkg/mgmt/lvmnode/lvmnode.go

```diff
@@ -54,13 +56,18 @@ func (c *NodeController) syncHandler(key string) error {
 // LVMNode
 func (c *NodeController) syncNode(namespace string, name string) error {
 	// Get the node resource with this namespace/name
-	cachedNode, err := c.NodeLister.LVMNodes(namespace).Get(name)
+	cachedNode, err := c.NodeLister.Namespace(namespace).Get(name)
```

Comment:
Can we name this variable more appropriately?

Reply:
This is the function dynamic lister exposes. This cannot be changed.

Reply:
**e.g.**

cachedNode --> cachedObj

Reply:
Name seems appropriate as it is -- cachedNode ✔️ .